src/components/footer/Footer.js

```diff
@@ -1,18 +1,13 @@
 import "./Footer.css";
 
 function Footer() {
-  const gryffindor = "Gryffindor".toUpperCase();
-  const ravenclaw = "Ravenclaw".toUpperCase();
-  const hufflepuff = "Hufflepuff".toUpperCase();
-  const slytherin = "Allies".toUpperCase();
-
   return (
     <footer>
       <nav id="house-filter">
-        <button className="nav-button-gryffindor">{gryffindor}</button>
-        <button className="nav-button-ravenclaw">{ravenclaw}</button>
-        <button className="nav-button-hufflepuff">{hufflepuff}</button>
-        <button className="nav-button-slytherin">{slytherin}</button>
+        <button className="nav-button-gryffindor">G</button>
+        <button className="nav-button-ravenclaw">R</button>
+        <button className="nav-button-hufflepuff">H</button>
```

Comment:
Maybe add `aria-label`s here? Maybe `G`, `R`, `H` are a bit too short to understand right away?

Reply:
Thank you! ❤️  Changed it and you will find a navbar with aria-labels in my next pull request. Wasn't sure about appropriate names. Can you maybe check and give me feedback?

Reply:
I will! .... tomorrow though 😉

Reply:
No hurry! I have to create the pull request first. I look forward to your feedback tomorrow. :-)